process/datadog_checks/process/data/conf.yaml.example

```diff
@@ -71,9 +71,9 @@ instances:
     ## @param thresholds - object - optional
     ## The threshold paramter is composed of two ranges: critical and warning
```

Comment:
```suggestion
    ## The threshold parameter is composed of two ranges: critical and warning
```

Reply:
> PR looks good. I'm thinking we might want to replace `-1` by `inf`, this would be clearer to understand when writing the config, and would reduce some logic in `process.py`. Do you think that'd work?

Actually `.inf` and `-.Inf` seems to be valid yaml float values, so maybe no change is needed.

```
Language Independent Scalar types:
    { ~, null }              : Null (no value).
    [ 1234, 0x4D2, 02333 ]   : [ Decimal int, Hexadecimal int, Octal int ]
    [ 1_230.15, 12.3015e+02 ]: [ Fixed float, Exponential float ]
    [ .inf, -.Inf, .NAN ]    : [ Infinity (float), Negative, Not a number ]
    { Y, true, Yes, ON  }    : Boolean true
    { n, FALSE, No, off }    : Boolean false
    ? !!binary >
        R0lG...BADS=
    : >-
        Base 64 binary value.
```

source https://yaml.org/refcard.html

Reply:
```python
>>> data = yaml.load("pos_inf: .inf\nneg_inf: -.Inf")
>>> pos_inf, neg_inf = data["pos_inf"], data["neg_inf"]
>>> (type(pos_inf), type(neg_inf))
(<class 'float'>, <class 'float'>)
>>> (pos_inf, neg_inf)
(inf, -inf)
```

Yup, looks good. We can probably just add documentation in the conf.yaml file

Reply:
Didn't know about that 👍